pyprotista/parsers/ident/msgfplus_2021_03_22_parser.py

```diff
@@ -117,43 +37,126 @@ def check_parser_compatibility(cls, file):
 
```

Comment:
Is file actually a str or a Path object?
If both is possible, adapt doc string and cast to Path if its a str